charts/kyverno/values.yaml

```diff
@@ -31,7 +31,7 @@ testImage:
   # testImage.repository defaults to "busybox" if omitted
   repository:
   # testImage.tag defaults to "latest" if omitted
-  tag:
+  tag: latest
```

Comment:
Setting the default test tag to `latest`, not sure if we want to custom this tag (or even the image).

Reply:
This won't work, `sed` will replace the tag anyways.